lib/wit/deps/http/types.wit

```diff
@@ -0,0 +1,570 @@
+/// This interface defines all of the types and methods for implementing
+/// HTTP Requests and Responses, both incoming and outgoing, as well as
+/// their headers, trailers, and bodies.
```

Comment:
I'm assuming files like this come from a standards body, and have been externally reviewed?

Reply:
`:nod:`